src/docs/customize/theming.mdx

```diff
@@ -422,13 +328,236 @@ Example:
 }}
 </Playground>
 
-### Form Layout
+### Check Fields
+
+Options shared by checkable form controls. This includes
```

Comment:
I would change `checkable` to `boolean`.

Reply:
It was under consideration by me, but:

- `boolean` is data type while my point of view is more about design/appearance (refer to `box` fields instead of `text` fields.
- `boolean` would be misleading for `Radio` – its value is rather _checkable_ than `boolean`.

Reply:
OK, I concur